pkg/module/go_get_fetcher.go

```diff
@@ -91,7 +91,7 @@ func getSources(goBinaryName string, fs afero.Fs, gopath, repoRoot, module, vers
 	disableCgo := "CGO_ENABLED=0"
 	enableGoModules := "GO111MODULE=on"
 
-	cmd := exec.Command(goBinaryName, "get", fullURI)
+	cmd := exec.Command(goBinaryName, "mod", "download", fullURI)
```

Comment:
@marwan-at-work we won't have to do all the gymnastics to create the `main.go` and the `go.mod` files anymore, right?

Reply:
@arschles the answer is unfortunately still no :)